scitasexternal/packages/amber/package.py

```diff
@@ -35,33 +34,43 @@ class Amber(Package):
     """
 
     homepage = 'http://ambermd.org'
-    url = 'file://fake/amber16.tgz'
+    url = 'file:///fake/path/to/amber-16.tgz'
 
     version('16', '652e24512146e966a0a50335572ebd66')
+    version('18', 'ae5438d3f1e2d3379d5664c34193ae48')
```

Comment:
Minor request for change. Every other package in Spack keeps the versions ordered from the most to the least recent. Can we swap the two lines above?